app/uk/gov/hmrc/estates/controllers/transformers/register/DeceasedTransformationController.scala

```diff
@@ -54,6 +56,44 @@ class DeceasedTransformationController @Inject()(
       }
   }
 
+  def getDateOfDeath: Action[AnyContent] = identify.async {
+    request =>
+      val result = transformationService.getTransformedData(request.identifier) map {
+        case Some(ComposedDeltaTransform(transforms: Seq[DeltaTransform])) =>
+          transforms.flatMap {
+            case DeceasedTransform(deceased) => Some(Json.toJson(deceased.dateOfDeath))
+            case _ => None
+          }.lastOption
+        case _ => None
+      }
+      result.map {
+        case Some(json) => Ok(json)
+        case None => Ok(Json.obj())
+      }
+  }
+
+  def getIsTaxRequired: Action[AnyContent] = identify.async {
+    request =>
+      val result = transformationService.getTransformedData(request.identifier) map {
+        case Some(ComposedDeltaTransform(transforms: Seq[DeltaTransform])) =>
+          transforms.flatMap {
+            case DeceasedTransform(deceased) =>
+
+              def withinCurrentTaxYear(dateOfDeath: LocalDate): Boolean =
+              (dateOfDeath.isEqual(LocalDate.of(LocalDate.now().getYear, 4,6)) || dateOfDeath.isAfter(LocalDate.of(LocalDate.now().getYear, 4,6))) &&
+                (dateOfDeath.isBefore(LocalDate.of(LocalDate.now().getYear + 1, 4,5)) || dateOfDeath.isEqual(LocalDate.of(LocalDate.now().getYear + 1, 4,5)))
+
```

Comment:
Could we store the month and days as config values to avoid use of magic numbers? It's not readily apparent what the 5th and 6th April refer to

Reply:
@BriWak could we not look to include this as a dependency as use it? https://github.com/hmrc/tax-year